.github/dependabot.yml

```diff
@@ -1,11 +0,0 @@
-# https://docs.github.com/code-security/dependabot/working-with-dependabot/dependabot-options-reference
-version: 2
-updates:
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "monthly"
-      time: "10:00"
-      timezone: "Asia/Tokyo"
-    assignees:
-      - "masutaka"
```

Comment:
@masutaka Please don't delete it, just change to me @shige!

Reply:
@shige I've made a change. I believe requesting a review is more appropriate than assigning, so I’ve updated the configuration in the CODEOWNERS file.